tools/extract_feature.cpp

```diff
@@ -199,6 +199,10 @@ main (int argc, char** argv)
   else if (feature_name == "VFHEstimation")
     computeFeatureViaNormals< VFHEstimation<PointXYZ, Normal, VFHSignature308>, PointXYZ, Normal, VFHSignature308>
     (cloud, output, argc, argv, false);
+  else {
```

Comment:
By convention we always break line before and after the open bracket symbol, unless it's a one liner.

Reply:
That makes sense, updated, thanks.